src/common/BAD_ICONS.js

```diff
@@ -0,0 +1,16 @@
+// icons in this array get our icons instead of the ones from blizzards API
+// make sure to add the correct icon in public/img/Icons/
```

Comment:
Could add a development time only check in this file to ensure the files exist. In fact importing the files could achieve this too and allow the images to be in a dir next to this file, which would be slightly cleaner.